mail/mlmmj/Portfile

```
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           codeberg 1.0
+PortGroup           legacysupport 1.1
+
+codeberg.setup      mlmmj mlmmj RELEASE_1_4_6
+
+name                mlmmj
+version             1.4.6
+categories          mail
+license             MIT
+maintainers         @inucat \
+                    proton.me:inucat
```

These two contact methods should be grouped:

```suggestion
maintainers         {@inucat proton.me:inucat}
```

Is it ok if others make minor modifications to your port without consulting you? If so, please add `openmaintainer`:

```suggestion
maintainers         {@inucat proton.me:inucat} openmaintainer
```

---

Now added `openmaintainer` here